presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -195,6 +195,9 @@ Property Name                                      Description
                                                    S3SelectPushdown.
 
 ``hive.metastore.load-balancing-enabled``          Enable load balancing between multiple Metastore instances
+
+``hive.skip-empty-files``                          Enable skipping empty files. Otherwise, it will produce an   ``false``
```

Comment:
a false
explicitly specify the default value of this property
What is a "false error"?

Reply:
screenshot of local docs build - the doc when built may answer all of these issues 

<img width="818" alt="Screenshot 2024-05-13 at 11 59 57 AM" src="https://github.com/prestodb/presto/assets/7013443/ea0dd770-7b7d-483c-8661-164ee3ed5902">

Reply:
Ah, it's a table.